plugin.rb

```diff
@@ -567,12 +567,15 @@ def self.denormalized_columns
 
   # Reimplement a couple ActiveRecord methods, but use PluginStore for storage instead
   class DataExplorer::Query
-    attr_accessor :id, :name, :description, :sql
+    attr_accessor :id, :name, :description, :sql, :user, :time
```

Comment:
Instead of `user` and `time` we should use `created_by` and `created_at` for clarity.